devito/function.py

```diff
@@ -536,26 +551,45 @@ def _halo_indices(self):
         """Return the function indices for which a halo region is defined."""
         return self.indices
 
+    @property
+    def derivatives(self):
+        """Return the list of available derivatives shortcuts"""
+        return self._derivatives + ["div", "laplace", "laplace2"]
```

Comment:
Why don't we put div, Laplace and laplace2 in ``initialize_derivatives`` as well?

Reply:
`laplace2` is a function, not a property, I guess the other two can.

Reply:
Looks like we (maybe just I) can't. These div, laplace need to access `self.dx`, .... and it seems to break everything if done inside `initialize_derivatives` (just try `print(self.dx)` after setting it up )